forge-gui/res/cardsfolder/upcoming/sproutwatch_dryad.txt

```diff
@@ -0,0 +1,8 @@
+Name:Sproutwatch Dryad
+ManaCost:1 G G
+Types:Creature Dryad
+PT:2/3
+T:Mode$ Phase | Phase$ BeginCombat | TriggerZones$ Battlefield | Execute$ TrigPump | TriggerDescription$ At the beginning of each combat, Sproutwatch Dryad gains flying until end of turn if a creature you control or a card in your hand has flying. The same is true for first strike, double strike, deathtouch, haste, hexproof, indestructible, lifelink, menace, reach, trample, and vigilance.
+SVar:TrigPump:DB$ PumpAll | ValidCards$ Creature.YouCtrl | KW$ First Strike & Flying & Deathtouch & Double Strike & Haste & Hexproof & Indestructible & Lifelink & Menace & Reach & Trample & Vigilance | SharedKeywordsZone$ Hand | SharedRestrictions$ Creature.YouOwn
```

Comment:
Shouldn't ValidCards just be Self?
I think ShardRestriction has to be YouCtrl again for the same reason as the other